src/instana/instrumentation/kafka/confluent_kafka_python.py

```diff
@@ -1,19 +1,27 @@
 # (c) Copyright IBM Corp. 2025
 
+
 try:
+    import contextvars
     from typing import Any, Callable, Dict, List, Optional, Tuple
 
     import confluent_kafka  # noqa: F401
     import wrapt
     from confluent_kafka import Consumer, Producer
+    from opentelemetry import context, trace
     from opentelemetry.trace import SpanKind
 
     from instana.log import logger
     from instana.propagators.format import Format
+    from instana.singletons import get_tracer
     from instana.util.traceutils import (
         get_tracer_tuple,
         tracing_is_off,
     )
+    from instana.span.span import InstanaSpan
+
+    consumer_token: Dict[str, Any] = {}
```

Comment:
Is this variable necessary to be a dictionary?  I saw that only one value to the unique key `token` is assigned to it, so I think a simple variable is sufficient, don't you think?

BTW, we are not using type annotations for variables in our code. 

```suggestion
    consumer_token = None
```